topo/node/cisco/cisco.go

```diff
@@ -315,31 +369,16 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 	if pb.Config.EntryCommand == "" {
 		pb.Config.EntryCommand = fmt.Sprintf("kubectl exec -it %s -- bash", pb.Name)
 	}
-	interfaceList, interfaceMap, err := generateInterfacesEnvs(pb)
-	if err != nil {
-		return nil, err
-	}
-	if pb.Config.Env == nil {
-		pb.Config.Env = map[string]string{
-			"XR_INTERFACES":                  interfaceMap,
-			"XR_CHECKSUM_OFFLOAD_COUNTERACT": interfaceList,
-			"XR_EVERY_BOOT_CONFIG":           filepath.Join(pb.Config.ConfigPath, pb.Config.ConfigFile),
+	if pb.Model == "xrd" {
+		err := setXRDEnv(pb)
+		if err != nil {
+			return nil, err
 		}
 	} else {
```

Comment:
Should there be an error is pb.Model is unexpected (neither "xrd" or "e8000"),

if so use a switch here:

switch pb.Model {
default:
  return nil, fmt.Errorf("unexpected model %q", pb.Model)
case "xrd":
  if err := setXRDEnv(pb); err != nil {
    return nil, err
  }
case "e8000":
  if err := setE8000Env(pb); err != nil {
    return nil, err
  }
}

either way to else is unnecessary and the err checks can be merged inline